.github/workflows/Student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Student < Person
```

Comment:
- [ ] Please fix the naming here the person file that you have created start with the capital letter Person.rb
![image](https://user-images.githubusercontent.com/74744787/216075365-401b9a7c-51e2-4416-92d8-3758e242ae77.png)